_specs_/websocket/schemas/crypto/events/aggregates.yml

```
@@ -0,0 +1,20 @@
+type: object
+properties:
+  ev:
+    type: string
+    enum:
+      - XA
+  pair:
+    $ref: '../../index.yml#/CryptoPair'
+  o:
+    $ref: '../../index.yml#/AggsOpen'
+  c:
+    $ref: '../../index.yml#/AggsClose'
+  h:
+    $ref: '../../index.yml#/AggsHigh'
+  l:
+    $ref: '../../index.yml#/AggsLow'
+  v:
+    $ref: '../../index.yml#/AggsVolume'
+  s:
+    $ref: '../../index.yml#/Timestamp'
```

Aggs also have an `e` attribute for the end time, not sure how to differentiate the meaning in the docs though

```suggestion
  s:
    $ref: '../../index.yml#/Timestamp'
  e:
    $ref: '../../index.yml#/Timestamp'
```